src/neuroconv/datainterfaces/behavior/sleap/sleap_utils.py

```diff
@@ -0,0 +1,25 @@
+from ....tools import get_package
+from ....utils import FilePathType
+
+
+def extract_timestamps(video_file_path: FilePathType) -> list:
+    """Extract the timestamps using pyav
+
+    Parameters
+    ----------
+    video_file_path : FilePathType
+        The path to the multimedia video
+
+    Returns
+    -------
+    list
+        The timestamps
+    """
+    av = get_package(package_name="av")
+
+    container = av.open(str(video_file_path))
+    stream = container.streams.video[0]
+    timestamps = [frame.time for frame in container.decode(stream)]
+    container.close()
```

Comment:
```suggestion
    with av.open(str(video_file_path)) as container:
        stream = container.streams.video[0]
        timestamps = [frame.time for frame in container.decode(stream)]
```

Reply:
@h-mayorquin Does this context work as suggested?

Reply:
Apparently, yes. I did not used it as a context manager at the beginning because it is not present in their documentation but after some further searching I found it: